### ISSUE TITLE
dEQP-VK.glsl.shader_clock.*: Assertion `scope == spv::ScopeDevice || scope == spv::ScopeWorkgroup' failed

### ISSUE THREAD

**Issue by Flakebi** [GPUOpen-Drivers/llpc#1009]
Running dEQP-VK.glsl.shader_clock.vertex.clockARB or other tests in dEQP-VK.glsl.shader_clock.* fails when assertions are enabled (`-DXGL_ENABLE_ASSERTIONS=ON`):
```python
Test case 'dEQP-VK.glsl.shader_clock.vertex.clockARB'..
deqp-vk: compiler/llpc/translator/lib/SPIRV/SPIRVReader.cpp:2895: llvm::Value *SPIRV::SPIRVToLLVM::transValueWithOpcode(SPIRV::SPIRVValue *const): Assertion `scope == spv::ScopeDevice || scope == spv::ScopeWorkgroup' failed.
```

**Comment by amdrexu** [comment]
@ruiling This might be relevant to https://github.com/GPUOpen-Drivers/llpc/pull/1001

**Comment by ruiling** [comment]
yes, I think this issue should be fixed by #1001.

**Comment by Flakebi** [comment]
You’re right, my llpc was outdated. Sorry for the noise.
